packages/mobile/src/screens/chats-screen/ChatsScreen.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react'
+
+import { chatActions, chatSelectors } from '@audius/common'
+import { View } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { Screen, Text } from 'app/components/core'
+
+export const ChatsScreen = () => {
+  const dispatch = useDispatch()
+  const chats = useSelector(chatSelectors.getChats)
+  const id = 'clbyghuy800003bat0th2ivy5'
+  const messages =
+    useSelector((state) => chatSelectors.getChatMessages(state, id)) ?? []
+
+  useEffect(() => {
+    dispatch(chatActions.fetchMoreChats())
+  }, [dispatch])
+
+  useEffect(() => {
+    dispatch(chatActions.fetchNewChatMessages({ chatId: id }))
+  }, [dispatch])
+
+  return (
+    <Screen>
+      <View>
+        <Text>You have {chats?.length} chats:</Text>
```

Comment:
?? 0 maybe? since it could be undefined chats?

Reply:
yeah this will change for sure